content/guides/dashboard/github-integration.md

```diff
@@ -283,6 +283,11 @@ steps:
 3. Click **Configure** beside the Cypress app.
 4. Click **Uninstall** below the "Uninstall Cypress" section.
 
+## Troubleshooting
+
+See
```

Comment:
Does this need to be tracked with an 'on' link? If not you can just add the relative path. something like...

`[Common Problems and Solutions](/guides/continuous-integration/github-actions#Common-problems-and-solutions)`